controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    vel_controller:
      type: velocity_controllers/JointGroupVelocityController
      joints: [gripper_joint]
      interface_name: velocity
      command_interfaces: [velocity]
      state_interfaces: [position, velocity]
      
    pos_controller:
      type: position_controllers/JointGroupPositionController
      joints: [gripper_joint]
      interface_name: position
      command_interfaces: [position]
      state_interfaces: [position]